### DISCUSSION TITLE
Radioking TX18S bootloader

### DISCUSSION THREAD

**Discussion by GeorgeMck15** [EdgeTX/edgetx#1943] [Q&A]
Hoping for some guidance with a new Radioking TX18S that I just purchased. It was delivered with Edge 2.4 with all the menu characters in some sort of Chinese symbols with a spattering of English just for good measure. After clicking on what may have been an Edge bin file it started a reboot and that seems to have black screened it. Using Companion it won't connect to any of my computers and there is no SD card showing either. When I try and turn it on it just flashes the red lights for a few seconds and then turn off. 
Is there any hope for it or is it a bin job?

**Comment by raphaelcoeffic** [comment]
@GeorgeMck15 head to https://buddy.edgetx.org and load the firmware for Jumper TX18, that should do the job.

**Reply by GeorgeMck15** [reply]
Thanks for the reply. I have got it up and running with Edge 2.5. I found a YouTube from a "Joshua Bardwell" on how to re-flash any of the Open source radios and followed his instructions and got it fixed. Just got to learn how to use it now and decide if I like it more than OpenTx.
